kubejs/server_scripts/normal/recipes/emendatusenigmatica/rods.js

```diff
@@ -0,0 +1,26 @@
+ServerEvents.recipes((event) => {
+    if (global.isNormalMode == false) {
+        return;
+    }
+    
+    const recipes = [];
+
+    let props = Object.keys(metal_properties);
+
+    props.forEach((metal) => {
+        if (Item.exists(`emendatusenigmatica:${metal}_rod`)) {
+            recipes.push({
+                output: Item.of(`emendatusenigmatica:${metal}_rod`, 4),
+                pattern: [' A ', ' A '],
```

Comment:
```suggestion
                pattern: ['A', 'A'],
```
The extra whitespace means the items have to be in the center